client/packages/invoices/src/OutboundShipment/DetailView/columns.ts

```diff
@@ -69,6 +73,8 @@ export const useOutboundColumns = () => {
         id: 'itemUnit',
         accessorKey: 'item.unitName',
         header: t('label.unit-name'),
+        enableColumnFilter: true,
+        filterVariant: 'select',
```

Comment:
I thought I already did this last week 🤔

Reply:
I think this diff actually is from merging your changes... 🤷